scripts/python-maintainer/makefile_template.jinja2

```diff
@@ -0,0 +1,35 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:={{ openwrt_name }}
+PKG_VERSION:={{ version }}
+PKG_RELEASE:=1
+
+PKG_MAINTAINER:={{ maintainer }}
+PKG_LICENSE:={{ license }}
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_SOURCE:={{ filename }}
+PKG_SOURCE_URL:=https://pypi.python.org/packages/source/{{ first_char }}/{{ package }}/
+PKG_HASH:={{ sha256 }}
+PKG_BUILD_DIR:=$(BUILD_DIR)/{{ package }}-$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/$(PKG_NAME)
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:={{ openwrt_name }}
```

Comment:
For example, package name is Unidecode
and I would prefer if the Title will be ASCII transliterations of Unicode text
The same as it is on https://pypi.org/project/Unidecode/

Reply:
I believe the `summary` field of the package data is a one-sentence description of the package. That would be a good starting point for `TITLE`.